test/end-to-end/testFunctionApp/HttpTriggerESModules/function.json

```diff
@@ -0,0 +1,21 @@
+{
+  "disabled": false,
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
+        "post"
+      ]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"
+    }
+  ],
+  "scriptFile": "./index.mjs"
```

Comment:
This is required for now - need to make [change in host](https://github.com/Azure/azure-functions-host/blob/dd062a5cc9e1f8e8bb875d3e6068b1df3865257a/src/WebJobs.Script/Host/FunctionMetadataProvider.cs#L245)